### PR TITLE
add zora uniswap v2 into dex.trades

### PR DESCRIPTION
**Pull request by andrewhong5297**
wow macros are great 👍 

adding zora trades

### CODE REVIEW

**Comment by andrewhong5297** [comment]
Waiting on decoded tables

File: sources/uniswap/zora/uniswap_zora_sources.yml
Comment (Hosuke) [inline sources/uniswap/zora/uniswap_zora_sources.yml:1]: We can add the sources here:
https://github.com/duneanalytics/spellbook/blob/main/sources/_sector/dex/trades/zksync/_sources.yml

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @andrewhong5297 

[Sample uniswap_v2_zora_base_trades]()

File: sources/uniswap/zora/uniswap_zora_sources.yml
Comment (Hosuke) [reply sources/uniswap/zora/uniswap_zora_sources.yml:1]: Sorry, you are right.

Comment (Hosuke) [reply sources/uniswap/zora/uniswap_zora_sources.yml:1]: Maybe `sources/_sector/dex/trades/zksync/_sources.yml` in future.

File: tokens/models/tokens/tokens_native.sql [hunk 1]
```diff
@@ -16,7 +16,7 @@ FROM (VALUES
          , ('arbitrum', 'ETH', 'WETH', 0x82af49447d8a07e3bd95bd0d56f35241523fbab1, 18)
          , ('avalanche_c', 'AVAX', 'WAVAX', 0xb31f66aa3c1e785363f0875a1b74e27b85fd66c7, 18)
          , ('gnosis', 'xDAI', 'WXDAI', 0xe91d153e0b41518a2ce8dd3d7944fa863463a97d, 18)
-         , ('bnb', 'BNB', 'WBNB', 0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c, 8)
+         , ('bnb', 'BNB', 'WBNB', 0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c, 18)
```
Comment (jeff-dude) [inline tokens/models/tokens/tokens_native.sql:19]: good catch

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
> wow macros are great 👍

that's the goal, glad to hear!

**Pull request merged**